torch/_tensor_docs.py

```diff
@@ -1457,11 +1457,11 @@ def callable(a, b) -> number
                r"""
 narrow_copy(dimension, start, length) -> Tensor
 
-Same as :meth:`Tensor.narrow` except returning a copy rather
-than shared storage.  This is primarily for sparse tensors, which
-do not have a shared-storage narrow method.  Calling ```narrow_copy``
-with ```dimemsion > self._sparseDims()``` will return a copy with the
-relevant dense dimension narrowed, and ```self.shape``` updated accordingly.
+Same as :func:`torch.narrow` except that :meth:`~FloatTensor.narrow_copy` is a Tensor method
+and it returns a copy rather than shared storage. This is primarily for sparse tensors, which
+do not have a shared-storage narrow method. Calling :meth:`~FloatTensor.narrow_copy`
+with ``dimemsion > self._sparseDims()`` will return a copy with the
```

Comment:
So.. if `dimemsion <= self._sparseDims()` it actually returns a shared storage tensor?

Reply:
Then this is misleading. Can we just delete this last sentence?

Reply:
here it says 'return a copy', maybe it is ok to keep it? I think it is helpful to mention the behavior of narrowing and copy dense tensor somewhere.